samples/cpp/greedy_causal_lm/greedy_causal_lm.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2023-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "openvino/genai/llm_pipeline.hpp"
+
+int main(int argc, char* argv[]) try {
+    if (3 > argc || argc > 4)
+        throw std::runtime_error(std::string{"Usage: "} + argv[0] + " <MODEL_DIR> \"<PROMPT>\" <DEVICE>");
+
+    std::string model_path = argv[1];
+    std::string prompt = argv[2];
+    
+    // GPU can be used as well
+    std::string device = "CPU";  
+    if (argc > 3) device = argv[3];
+
+    ov::genai::LLMPipeline pipe(model_path, device);
+    ov::genai::GenerationConfig config = pipe.get_generation_config();
+    config.max_new_tokens = 100;
+    config.do_sample = false;
+    auto streamer = [](std::string subword){ std::cout << subword << std::flush; return false; };
```

Comment:
do we need to demonstrate streaming here? 
I suppose `chat_sample` is enough for it

Reply:
removed from here, left streamer only in chat sample